tests/unit/cmd/iac/test_scan_all.py

```diff
@@ -244,15 +245,15 @@ def test_iac_scan_all_verbose(cli_fs_runner: CliRunner, cli_command) -> None:
 
 @patch("pygitguardian.client.GGClient.iac_directory_scan")
 def test_iac_scan_all_ignored_directory(
-    iac_directory_scan_mock: Mock, cli_fs_runner: CliRunner, cli_command
+    iac_directory_scan_mock: Mock, cli_fs_runner: CliRunner, cli_command, tmp_path: Path
 ) -> None:
     """
     GIVEN a directory which is ignored
     WHEN running the iac scan all command on this directory
     THEN an error is raised
     """
-    path = Path(".")
-    repo = Repository.create(path)
+    path = Path(tmp_path)
```

Comment:
Since `tmp_path` is already a `Path`, I think you can just drop this line and use `tmp_path` below.